src/runmode-af-packet.c

```diff
@@ -738,7 +738,9 @@ int AFPRunModeIsIPS()
     }
 
     if (has_ids && has_ips) {
-        SCLogInfo("AF_PACKET mode using IPS and IDS mode");
+        SCLogWarning(SC_ERR_INVALID_ARGUMENT,
+                "AF_PACKET using both IPS and TAP/IDS mode, this will not "
+                "be allowed in Suricata 8 due to undefined behavior.");
```

Comment:
I think it makes sense to reference the ticket id, so we can add more info after the fact or address questions.

Reply:
Full link?  Or Just the "See ticket #xxxx for more details.".

Reply:
Ah, the latter I guess. As this seems to be an existing pattern in the code.